src/components/Navbar.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import styled from 'styled-components';
+import { NavLink } from 'react-router-dom';
+
+const Navbar = () => {
+  return (
+    <NavContainer>
+      <LinkTo to="/">
```

Comment:
You are nesting many Elements inside divs. Not sure if that is really necessary...
As an alternative you could do something like this
`<button>`
   ` <Link>`
     ` <img />`
  ` </Link>`
`</button>`